client/components/action-card/index.jsx

```diff
@@ -27,10 +28,11 @@ const ActionCard = ( {
 	buttonOnClick,
 	children,
 	compact,
+	classNames,
```

Comment:
It's a new property, so let's add it to its README: https://github.com/Automattic/wp-calypso/blob/master/client/components/action-card/README.md

Reply:
Fixed in [7f47c5f](https://github.com/Automattic/wp-calypso/pull/34956/commits/7f47c5f122ad381937198b04c08436041ecb05ba)